### PR TITLE
Update Quickstarts 1-3 for ASP.NET Core 3

### PR DESCRIPTION
**Pull request by leastprivilege**

### CODE REVIEW

File: docs/quickstarts/1_client_credentials.rst [hunk 1]
```diff
@@ -197,17 +188,25 @@ Update `Startup` to look like this::
...
+            });
         }
     }
 
 
-``AddAuthentication`` adds the authentication services to DI and configures ``"Bearer"`` as the default scheme. 
-``UseAuthentication`` adds the authentication middleware to the pipeline so authentication will be performed automatically on every call into the host.
+* ``AddAuthentication`` adds the authentication services to DI and configures ``Bearer`` as the default scheme. 
+* ``UseAuthentication`` adds the authentication middleware to the pipeline so authentication will be performed automatically on every call into the host.
+* ``UseAuthorization`` adds the authorization middleware to make sure, our API endoint cannit be accessed by anonymous clients.
```
Comment (srnkrkgrd) [inline docs/quickstarts/1_client_credentials.rst]: typo on this line

> our API enoint cannit

our API endpoint can not

File: samples/Quickstarts/2_InteractiveAspNetCore/src/IdentityServer/Program.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
                             .MinimumLevel.Override("System", LogEventLevel.Warning)
                             .MinimumLevel.Override("Microsoft.AspNetCore.Authentication", LogEventLevel.Information)
                             .Enrich.FromLogContext()
-                            .WriteTo.File(@"identityserver4_log.txt")
+                            //.WriteTo.File(@"identityserver4_log.txt")
```
Comment (srnkrkgrd) [inline samples/Quickstarts/2_InteractiveAspNetCore/src/IdentityServer/Program.cs]: Zombie code as Serilog.Sinks.File isn't included in packagereference. Any reason to keep it?

File: docs/quickstarts/1_client_credentials.rst (hunk 1 above)
Comment (leastprivilege) [reply docs/quickstarts/1_client_credentials.rst]: Thanks!

**Comment by genoher** [comment]
Why delete Sample 5? Hybrid flow?

**Pull request merged**